pkg/antctl/raw/check/installation/command.go

```diff
@@ -215,9 +215,9 @@ func (t *testContext) setup(ctx context.Context) error {
 			},
 		},
 		Tolerations: commonToleration,
-		Labels:      map[string]string{"app": echoSameNodeDeploymentName},
+		Labels:      map[string]string{"app": "antrea", "component": "installation-checker", "name": echoSameNodeDeploymentName},
```

Comment:
I didn't think about this earlier, but I think it may be a good idea to also annotate the test Namespace that we create with the correct `app` and `component` labels? Same thing for the `check cluster` command.
In particular, it would be useful if the test is interrupted and the user is left with 1 (or more, if this happens multiple times) test Namespace in their cluster.

Reply:
Yes. That would surely be very helpful for the user. thanks :)